base/doc/clsguide.tex

```diff
@@ -1181,6 +1181,113 @@ \subsection{Querying localisation}
 where |#1| is the user input and the first argument to
 \cs{MakeUppercaseAux} takes two arguments, the locale and input text.
 
+\subsection{Extended and expandable references of properties}
+
+A property is something that \LaTeX{} can track, such as a page number, a section number or a name. Such properties
+can be labeled, their current value written to the \texttt{aux}-file and referenced in the next compilation, similar to the standard \cs{label}/\cs{ref} commands. 
```

Comment:
and then referenced in the next compilation, similar to the standard \cs{label}/\cs{ref} commands (that record/reference a fixed set of properties: currentlabel,page,title, and target).

Reply:
done